### PR TITLE
add cpu info check for lam lass

### PR DESCRIPTION
**Pull request by gxl8483336**
check CPUID and flags for lam and lass

### CODE REVIEW

File: BM/instruction-check/feature_list.py [hunk 1]
```diff
@@ -145,6 +145,14 @@
         "cpuid": ['7', '0', '1', '0', 'a', '7'],
         "platforms": {"SRF", "CWF"}
     },
+    "lam": {
+        "cpuid": ['7', '0', '1', '0', 'a', '26'],
+        "platforms": {"SRF", "CWF"}
+    },
+    "lass": {
+        "cpuid": ['7', '0', '1', '0', 'a', '6'],
```
Comment (xhao22) [inline BM/instruction-check/feature_list.py:153]: Can you sort it by number? For example: ['7', '0', '1', '0', 'a', '6'] is on above of ['7', '0', '1', '0', 'a', '7'].

File: BM/instruction-check/feature_list.py [hunk 2]
```diff
@@ -145,6 +145,14 @@
         "cpuid": ['7', '0', '1', '0', 'a', '7'],
         "platforms": {"SRF", "CWF"}
     },
+    "lam": {
```
Comment (xhao22) [inline BM/instruction-check/feature_list.py:148]: All the feature name in this file are capital letter.

File: KVM/qemu/x86_cpuid.cfg [hunk 3]
```diff
@@ -19,6 +19,10 @@
             features = "AVX_NE_CONVERT"
         - cmpccxadd:
             features = "CMPCCXADD"
+        - lam:
+            features = "lam"
```
Comment (xhao22) [inline KVM/qemu/x86_cpuid.cfg:23]: "lam" capital letter here if you buy in my first comment in BM/instruction-check/feature_list.py

Comment (gxl8483336) [reply KVM/qemu/x86_cpuid.cfg]: changed to capital.

File: BM/instruction-check/feature_list.py (hunk 1 above)
Comment (gxl8483336) [reply BM/instruction-check/feature_list.py:145]: updated with sorting by CPUID number.

File: BM/instruction-check/feature_list.py (hunk 2 above)
Comment (gxl8483336) [reply BM/instruction-check/feature_list.py]: updated with capital letter for feature name.

**Pull request merged**